### ISSUE TITLE
Add User Error: The request has been black-holed

### ISSUE THREAD

**Issue by Applenice** [MISP/MISP#4950]
| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug, Question
| OS version (server)       | Ubuntu 18.04
| OS version (client)       | Win10, Ubuntu 18.04
| PHP version               | 7.2
| MISP version / git hash   | v2.4.111
| Browser                   | Chrome 75.0

### Expected behavior
normal work

### Actual behavior
An error occurred while adding a user and clicking Submit: `Error: An Internal Error Has Occurred`，But I can see in the user list that the user has been successfully added and can log in to MISP normally.

### Steps to reproduce the behavior
`https://192.168.0.2/admin/users/add`->add information->Click Submit->Prompt error.

### Logs, screenshots, configuration dump, ...
```
2019-08-02 15:50:33 Error: [BadRequestException] The request has been black-holed
Request URL: /admin/users/add
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Component/SecurityComponent.php(831): AppController->blackhole('secure')
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Component/SecurityComponent.php(351): SecurityComponent->_callback(Object(UsersController), 'blackHole', Array)
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Component/SecurityComponent.php(255): SecurityComponent->blackHole(Object(UsersController), 'secure', Object(SecurityException))
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Utility/ObjectCollection.php(129): SecurityComponent->startup(Object(UsersController))
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')
#5 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#6 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()
#7 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#8 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#9 {main}
```

**Comment by SteveClement** [comment]
@Applenice please test again, should be fixed in latest 2.4 branch. re-open if persistent with more details.

**Comment by Applenice** [comment]
I am very happy to receive your reply, I just verified it, it works fine, thank you.
